leonzwagner/my-portofolio
# 1 tailwindcss setup

### Description

This pull request initialize tailwindcss. It'll be using in styling thing.

### Issue

Adresses #1 (Feature enhancement with new dependencies)

### Steps to Test

1. Ensure that all existing features are functioning correctly.
2. Verify that the new dependencies are correctly installed:
   - Check `package.json` to see that `tailwindcss`, `postcss` and `autoprefixer` are listed under devDependencies.
3. Create new tsx and write down some tailwind utilites class to make sure it works correctly.

### Checklist

- [x] New dependencies are added to `package.json`

### Additional Notes

Please ensure that you run `npm install` or `pnpm install` to update your local dependencies.
